content/How to work with Stripe.md

```diff
@@ -0,0 +1,349 @@
+# **Online payments done SIMPLE - How to work with Stripe**
```

Comment:
I think I would use `made` here, instead of `done`. You make payments, instead of do payments.